### ISSUE TITLE
[Feature Request] modal-append-to-body for dropdown

### ISSUE THREAD

**Issue by ashwin0003** [ElemeFE/element#10290]
### Existing Component
Yes

### Component Name
dropdown

### Description
Looking a equivalent of modal-append-to-body prop for the dropdown component.

Currently the dropdown list is being appended to the end of the body.
I need a option to append the dropdown list to be parent dropdown.

My setup being
```html
<html>
  <body>
    <my-app>
    </my-app>
  </body>
</html>
```

Appending dropdown list at the end of the body is dis-regarding my custom css thats being added in the component 'my-app'.

**Comment by Leopoldthecoder** [comment]
I think adding a `popper-class` to `<el-dropdown-menu>` makes more sense in your use case.

**Comment by ashwin0003** [comment]
I'm not sure if I understand.

Looking at popper.js and vue-popper didn't ring any bells.

Are you perhaps suggesting I use element-ui's popover component?

**Comment by Leopoldthecoder** [comment]
https://jsfiddle.net/leopoldthecuber/yrhk6a9a/

Internally we have a mixin called `vue-popper`, which takes care of popup menus for all components. It has a prop `append-to-body`.

DropDown Menu also uses this mixin, so you can directly set `append-to-body` on it. Though you may need to override `position` of `.el-dropdown` so the style won't go crazy:
```css
.el-dropdown {
  position: static;
}
```

**Comment by ashwin0003** [comment]
Thank you. Solved.
And also thank you for the fast replies and resolution.

The prop `:append-to-body="false"`  helped with my issue. Didn't need to set el-dropdown's position to static but did it anyways just incase some other changes in the future mess things up.

I had tried `:modal-append-to-body="false"` but that didn't help.
Dev tools for vue didn't show `append-to-body` and `modal-append-to-body` in its list of props and it was neither in the docs so definitely wouldn't have tried the suggested solution.
